parsec/core/gui/parsec_application.py

```diff
@@ -91,7 +94,7 @@ def has_active_modal(cls):
         return False
 
     @classmethod
-    def get_main_window(cls):
+    def get_main_window(cls) -> QWidget | None:
```

Comment:
Cast the return value as `MainWindow` (unless there's an circular import problem)